### ISSUE TITLE
Incorrect limitations when using permissions to expand them

### ISSUE THREAD

**Issue by wellnesscookie** [BentoBoxWorld/Limits#29]
**Description**
A picture of our blocklimits config: https://i.imgur.com/qgSv4rW.png
A picture of our permissions for SkyCrafter rank: https://i.imgur.com/SvWlBvb.png
A picture of our /is limits display and a rank in the corner to prove incorrect settings:
http://prntscr.com/o6mwj5

**Steps to reproduce the behavior:**
- Have same configurations as above in order to reproduce

**Expected behavior**
Display and only allow the limit given via permission and not more than it

**Server Information:**

 - Database being used: mySQL
 - OS: Debian/ Linux
 - Java Version: Java 10
 - BBox version output:
```
[23:08:13] [main/INFO]: [CHAT] Running PAPER 1.13.2.
[23:08:13] [main/INFO]: [CHAT] BentoBox version: 1.5.2
[23:08:13] [main/INFO]: [CHAT] Loaded Game Worlds:
[23:08:13] [main/INFO]: [CHAT] acidisland (acidisland): Overworld, Nether, End
[23:08:13] [main/INFO]: [CHAT] skyblock (skyblock): Overworld, Nether, End
[23:08:13] [main/INFO]: [CHAT] Loaded Addons:
[23:08:13] [main/INFO]: [CHAT] AcidIsland 1.5.0 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] BentoBox-InvSwitcher 0.0.3-SNAPSHOT (ENABLED)
[23:08:13] [main/INFO]: [CHAT] Biomes 1.5.0.0 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] BSkyBlock 1.5.0 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] Challenges 0.7.5 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] DimensionalTrees 1.1 - SerbCraft (ENABLED)
[23:08:13] [main/INFO]: [CHAT] Level 1.5.0 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] Limits 0.2.0-SNAPSHOT (ENABLED)
[23:08:13] [main/INFO]: [CHAT] SerbCraftAddon 2.3 (ENABLED)
[23:08:13] [main/INFO]: [CHAT] WelcomeWarps 1.5.0 (ENABLED)
```

**Additional context**
- - Some seem to be fine, but beacon does not.

**Comment by tastybento** [comment]
I'm pretty sure this was fixed in 0.2.2. Could you try the latest release?

**Comment by wellnesscookie** [comment]
Okay, will test soon. It's peak time for our server at the moment.

**Comment by wellnesscookie** [comment]
Well, some of the limits seem to be fixed, but some just tend to be really weird.
This is our moderator on his alt account, you can see he has a SkyLord rank:
http://prntscr.com/o7wznn

Now, these are the permissions set:
https://i.imgur.com/AAYvRJu.png
And this is our configuration:
https://pastebin.com/ekS9uvpd (For common players)

SkyCrafter is shown a limit of 20, even tho he has a permission for 25, and SkyLord is shown 25, even tho he has a permission for 30.
We even tried unsetting the skycrafter's permission because skylord does inheritates skycrafter, but no, it's not due to it either.

**Comment by tastybento** [comment]
The limit maximums are set at the time of login. Could the player's permissions have changed since they logged in? If they log out and back in does it update?

**Comment by wellnesscookie** [comment]
No, they do not update. I have read that too somewhere and each time we changed a permission - the player relogged to test.

**Comment by tastybento** [comment]
Found the bug.

**Comment by tastybento** [comment]
Should be fixed in a886aaac1f847235cccd1464dcce3c58417ed10c

**Comment by wellnesscookie** [comment]
Works splendid!
